src/Utils/WordPress/files/php.ini

```diff
@@ -0,0 +1,9 @@
+google_app_engine.enable_functions = "php_sapi_name, gc_enabled"
```

Comment:
I've never seen these google_app_engine directives, is there documentation to which this could link?

Reply:
I think these may be relevant for `php55` only.

Reply:
There's nothing  in the [ini](http://php72-ini.appspot.com/) about these values. It's possible these are carried over from the `php55` version of this tutorial. I'll verify this.

Reply:
Additionally, there's nothing in the [php72 docs](https://cloud.google.com/appengine/docs/standard/php7/config/php_ini) about it either, but the [php55 docs](https://cloud.google.com/appengine/docs/standard/php/config/php_ini) have these values. So I will remove them and verify.

Good catch!